libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -393,6 +405,8 @@ bool AC_Fence::check_fence_alt_max()
 /// cause the altitude fence to be freshly breached
 bool AC_Fence::check_fence_alt_min()
 {
+    const float alt_min_relhome = get_alt_min_relhome();
```

Comment:
```suggestion
    const float alt_min_relhome = get_alt_min(Location::AltFrame::ABOVE_HOME);
```

Similarly elsewhere.